docs/api/app.md

```diff
@@ -565,9 +565,11 @@ the active app. On Windows, focuses on the application's first window.
 
 Hides all application windows without minimizing them.
 
-### `app.show()` _macOS_
+### `app.show(noActivate)` _macOS_
 
-Shows application windows after they were hidden. Does not automatically focus
+* `noActivate` String - whether or not app should activate windows after restoring them.
+
+Shows application windows after they were hidden. If `noActivate` is `true`, the app does not automatically focus
```

Comment:
this documentation was previously incorrect; our implementation did automatically focus the restored windows.